### PR TITLE
Document Falko API

### PR DESCRIPTION
**Pull request by FilipeKN4**
## Proposed Changes

Create Falko API documentation using the gem "apipie". 

Solve #211 

## Type of Change

What kind of changes this Pull Request introduces to Falko?
_`x` to what it applies_

- [ ] Bugfix
- [x] New Feature

## Checklist

_`x`to what it applies. If not sure about some topics, don't hesitate to ask. We're here to help!_

- [x] Pull Request's name is self-explanatory and its in English.
- [x] There are no _Rubocop_ issues.
- [x] Travis build has success (tests, code climate and codacy checks).
- [x] Pull Request is linked to an existing issue.
- [ ] Seeds have been created to the new User Stories.

### CODE REVIEW

File: config/initializers/apipie.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Apipie.configure do |config|
+  config.app_name                = "Falko20172"
+  config.api_base_url            = ""
+  config.doc_base_url            = "/apipie"
+  config.translate               = false
+  # where is your API defined?
```
Comment (MatheusRich) [inline config/initializers/apipie.rb]: Is this really needed?

**Comment by MatheusRich** [comment]
@FilipeKN4 Nice work here! So much thanks! Just one question: Is there another way to document this endpoints? I mean, as much as I like having this, I would like to not have them inside the controllers ('cause this makes them big and hard to read the code itself). Is there another file or something where we could put this docs?

File: config/initializers/apipie.rb (hunk 1 above)
Comment (FilipeKN4) [reply config/initializers/apipie.rb]: This is required to configure apipie and define a url to show all endpoints.

**Comment by FilipeKN4** [comment]
@MatheusRich I made the changes for the documentation stay out the controllers now.

**Comment by alaxalves** [comment]
@FilipeKN4 btw this is related to https://github.com/falko-org/Falko/issues/347. Nice work tho

File: config/initializers/apipie.rb (hunk 1 above)
Comment (MatheusRich) [reply config/initializers/apipie.rb]: I mean the comment `# where is your API defined?`.

**Comment by MatheusRich** [comment]
@FilipeKN4 Nicely done. One last question: where the docs can be accessed? is there some specific route?

**Comment by alaxalves** [comment]
> @FilipeKN4 Nicely done. One last question: where the docs can be accessed? is there some specific route?

@MatheusRich I think that by visiting `http://localhost:3000/apipie` according to [this](https://github.com/Apipie/apipie-rails). Right @FilipeKN4 ? BTW your commits are not linked to your GH user.

File: config/initializers/apipie.rb (hunk 1 above)
Comment (FilipeKN4) [reply config/initializers/apipie.rb]: Really wasn't necessary, I removed the comment.

**Comment by FilipeKN4** [comment]
> > @FilipeKN4 Nicely done. One last question: where the docs can be accessed? is there some specific route?
> 
> @MatheusRich I think that by visiting `http://localhost:3000/apipie` according to [this](https://github.com/Apipie/apipie-rails). Right @FilipeKN4 ? BTW your commits are not linked to your GH user.

It's right, the docs can be acessed in `http://localhost:3000/apipie`. I linked my GH user for next commits, was a mistake.

**Review by alaxalves — APPROVED** [review]

**Pull request merged**